app/styles/_header.scss

```diff
@@ -1 +1,66 @@
 // Styles for the header
+.search{
+ width: 25em;
+ height: 2.5em;
+ border-color: $grey;
+}
+
+#menu{
+  background-color: $turquoise;
+  text-align: center;
+  color: white;
+  font-family: Bernard MT Condensed;
+  font-size: 16px;
+  cursor:pointer;
+  height: 1.5em;
+  }
+
+#col1{
```

Comment:
All of the have the same style, please change the id for a class so you can easily update it